misc/userscripts/password_fill

```diff
@@ -209,6 +210,78 @@ choose_entry_zenity_radio() {
     choose_entry_menu || true
 }
 
+# Default backend selection.
+backend="pass"
```

Comment:
This value should be provided some other way because if this gets merged, this script will be in `/usr/share/qutebrowser/userscripts` and will get overwritten every time qutebrowser is updated (AFAIK).

Reply:
This value will be overwritten upon loading the `password_fill_rc` when the userscript is executed will it not? I added this as a default to execute the same functionality as prior to these changes. That is unless a user explicitly defines `backend=last_pass` in `password_fill_rc`. Or is there something else I am missing here?

Reply:
Yes, you're correct. I have no idea how this script works and I can't read. =/

Reply:
There is no reason to add these line. The way the script works is: every backend consists of three functions: `init`, `query_entries` and `open_entry`. In your `password_fill_rc` you simply call the backend and it will load itself, overwriting the existing backend, for example, for the `last_pass_backend`, just put `last_pass_backend` in your `password_fill_rc`, for the `pass_backend` just put `pass_backend` in there; there is no need for a variable `$backend`. Moreover, it is even harmful, because it prevents the user to define a (partially) custom backend in the `password_fill_rc` and use that.